servo-tidy.toml

```diff
@@ -39,6 +40,8 @@ num = [
 # Ignored packages with duplicated versions
 packages = [
   "bitflags",
+  "crossbeam-epoch",
+  "crossbeam-utils",
```

Comment:
I am adding these tidy dup exceptions in #21539, in addition to an `ipc-channel` exception that can be removed once this PR lands.

https://github.com/servo/servo/pull/21539#issuecomment-420216056